velox/type/Filter.cpp

```diff
@@ -490,35 +490,65 @@ std::unique_ptr<Filter> MultiRange::mergeWith(const Filter* other) const {
       return this->clone(/*nullAllowed=*/false);
     case FilterKind::kDoubleRange:
     case FilterKind::kFloatRange:
-    case FilterKind::kBytesRange:
-    case FilterKind::kBytesValues:
-      // TODO Implement
+      // TODO: Implement
```

Comment:
@mbasmanova  Added the function back to the right  location

Reply:
Thanks.